### ISSUE TITLE
[SETUP] Write a docker environment - Try running the app inside docker

### ISSUE THREAD

**Issue by zappingseb** [healthhackersER/CAMA-CORE-Example#36]
# Problem
_As a ``developer`` I can ``run the app inside a docker container``, so that ``access it on my phone``_

# Solution

- [ ] Write a Dockerfile & dockercompose.yml that work with our repo and expo-cli
  - [ ] Maybe mount our app by the `-v` command or find another solution, as the guy did
  - [x] Check out how running VSCode inside a container works
- [x] Run the guide which should be finished in #12 for docker & Linux
- [ ] Check if it works
- [ ] If not, change the guide

# Goal
- [ ] Have a Dockerfile + dockercompose + README.md to run a docker image with our stuff
- [ ] Have a Wiki Page with the description of where to find the stuff
- [ ] Have the app on your phone from a docker container

**Comment by IvoLeist** [comment]
If you would rephrase the problem to 
..., so that I can **access it on a virtual device** we could finish that milestone
combining: https://github.com/healthhackersER/CAMA-CORE-Example/issues/34 + https://github.com/healthhackersER/CAMA-CORE-Example/issues/12

Regarding how to access it on the phone if the app is running on a docker container
I am still struggling with it xD
